### PR TITLE
Add AWS Service Broker section

### LINKED ISSUE #170: Support for AWS service broker
https://github.com/awslabs/aws-servicebroker is now in public beta and Dev has been working to get CAP to work with it. We're looking at this for 1.3.

Initial documentation exists here: https://trello.com/c/oXSOW29r/778-5-new-aws-broker though QA still hasn't gone through it as of writing.

### PR DESCRIPTION
**Pull request by btat**
Resolves #170

### CODE REVIEW

File: xml/cap_depl_eks.xml [hunk 1]
```diff
@@ -513,4 +513,225 @@ secrets:
...
+    <step>
+     <para>
+      Create a server certificate for the service broker:
+     </para>
+     <substeps>
+      <step>
+       <para>
+        Get the CA certificate:
+       </para>
+       <screen>kubectl get secret --namespace scf --output jsonpath='{.items[*].data.internal-ca-cert}' | base64 -di > ca.pem</screen>
```
Comment (cjschroder) [inline xml/cap_depl_eks.xml]: Maybe suggest to do this in a separate directory, where it won't conflict with other ca.* files?

File: xml/cap_depl_eks.xml [hunk 2]
```diff
@@ -513,4 +513,225 @@ secrets:
...
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      The <link xlink:href="https://www.openssl.org/docs/man1.0.2/apps/openssl.html">OpenSSL command line tool</link>
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Ensure the user or role running the broker has an IAM policy set as specified <link xlink:href="https://github.com/awslabs/aws-servicebroker/blob/master/docs/install_prereqs.md#iam">here</link>
```
[the remark this answers is not in the document]
Comment (btat) [reply xml/cap_depl_eks.xml]: Fixed in https://github.com/SUSE/doc-cap/pull/289/commits/dca9e2098eb6f86bdb52bbc14c588667bc8163aa

File: xml/cap_depl_eks.xml (hunk 1 above)
Comment (btat) [reply xml/cap_depl_eks.xml]: Added in https://github.com/SUSE/doc-cap/pull/289/commits/dca9e2098eb6f86bdb52bbc14c588667bc8163aa

Comment (cjschroder) [reply xml/cap_depl_eks.xml]: awesome sauces!

**Pull request merged**